.vscode/settings.json

```diff
@@ -1,26 +1,35 @@
 {
-	"eslint.validate": [
-		{
-			"language": "javascript",
-			"autoFix": false
-		},
-		{
-			"language": "javascriptreact",
-			"autoFix": false
-		},
-		{
-			"language": "typescript",
-			"autoFix": false
-		},
-		{
-			"language": "typescriptreact",
-			"autoFix": false
-		}
-	],
 	"cSpell.words": [
 		"dismissable",
 		"ethnicities",
 	],
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
+	"[javascript]": {
+		"editor.codeActionsOnSave": {
+			"source.fixAll.eslint": true
+		}
+	},
+	"[javascriptreact]": {
+		"editor.codeActionsOnSave": {
+			"source.fixAll.eslint": true
+		}
+	},
+	"[typescript]": {
+		"editor.codeActionsOnSave": {
+			"source.fixAll.eslint": true
+		}
+	},
+	"[typescriptreact]": {
+		"editor.codeActionsOnSave": {
+			"source.fixAll.eslint": true
+		}
+	},
+	"eslint.codeAction.showDocumentation": {
```

Comment:
I would remove these empty eslint actions though. Make sure if you're customizing your IDE that you do it in your User/ Global settings and have it override the workspace.

Reply:
Good point!  I reverted this file.